File: leetcode2/0485. Max Consecutive Ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        cnt=0
        rep=0
        for num in nums:
            if num==1:
                cnt+=1
            else:
                rep=max(rep,cnt)
                cnt=0
        return max(rep,cnt)

File: C#/0485. Max Consecutive Ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int rep = 0;
        int cnt = 0;
        foreach (int num in nums)
        {
            if (num == 1)
            {
                cnt += 1;
            }
            else if(cnt>0)
            {
                rep = Math.Max(rep, cnt);
                cnt = 0;
            }
        }
        return Math.Max(rep, cnt);
    }
}
